### PR TITLE
34518

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -6,6 +6,83 @@ uid: DCP_change_log
 
 The DataMiner Cloud Platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 27 September 2022 – New feature – DataMiner Teams bot support for custom commands [ID_34518]
+
+If CoreGateway version 2.11.0 or higher and FieldControl version 2.8.1 or higher are installed (included in Cloud Pack version 2.8.0), the DataMiner Teams bot now allows you to display and run custom commands with dynamic user input configured in your cloud-connected DataMiner System.
+
+You can do so using the Teams bot commands *show command \<command name\>*, *show command*, and *run command \<command name\>*.
+
+To add a command to your DMS, create an Automation script in the folder "bot" in the DMS. For examples of such scripts, refer to [ChatOps Extensions](https://github.com/SkylineCommunications/ChatOps-Extensions) on GitHub.
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:15]: can refer directly to CustomCommandExamples https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/CustomCommandExamples

**Review by JordyGit — COMMENTED** [review]
In general most of these things could be in the docs. In the RN we should just tell them in short about the newly supported custom commands, refer to the docs, and where they can find some examples on getting started (github repo).
Also because we now have the github repo, the actual code examples could be left out and refer to the github?

File: release-notes/Cloud/DCP_change_log.md (hunk 1 above)
Comment (JordyGit) [reply release-notes/Cloud/DCP_change_log.md:15]: same for the other file

Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md]: Any further changes needed after my latest commit?

File: user-guide/Cloud_Platform/TeamsBot/DataMiner_Teams_bot.md [hunk 2]
```diff
@@ -90,3 +96,36 @@ You can use the following options to interact with the DataMiner Teams bot:
...
+## Adding commands for the Teams bot to a DMS
+
+To add a command for the Teams bot to your DMS, create an Automation script in the folder "bot" in the DMS.
+
+> [!TIP]
+> For examples of command scripts, refer to [Custom Command Examples](https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/CustomCommandExamples) on GitHub.
+
+### Input and output of the commands
+
+The commands allow dynamic input, such as dummies, parameters, parameters with value files, and memory files.
```
Comment (JordyGit) [inline user-guide/Cloud_Platform/TeamsBot/DataMiner_Teams_bot.md:109]: maybe we can refer to Automation docs for those inputs if those exist?

**Comment by JordyGit** [comment]
Do we leave all the info from the docs in the RN? IMO we can link to the docs & examples and make the RN only a few lines long?

**Comment by MariekeGO** [comment]
> Do we leave all the info from the docs in the RN? IMO we can link to the docs & examples and make the RN only a few lines long?

I'd prefer to leave the summarized information in the release note. This way, after there have been updates, people can still look back at the release notes to see how something was in a particular version (which can be useful for troubleshooting).

**Comment by JordyGit** [comment]
Ok, then it's more a change in mindset I have to do 😃

**Pull request merged**